### PR TITLE
refact training scales controller and add the corresponding tutorial

### PR DESCRIPTION
**Pull request by Joker316701882**
This PR may fix the potentially poor performance on custom datasets, especially those that change the image sizes. #521

### CODE REVIEW

**Comment by Joker316701882** [comment]
@ruinmessi @FateScript Please help review.

File: README.md [hunk 1]
```diff
@@ -186,7 +186,8 @@ python tools/eval.py -n  yolox-s -c yolox_s.pth -b 1 -d 1 --conf 0.001 --fp16 --
 <details>
 <summary>Tutorials</summary>
 
-*  [Training on custom data](docs/train_custom_data.md).
+*  [Training on custom data](docs/train_custom_data.md)
+*  [Controlling training image size](docs/control_training_image_size.md)
```
Comment (FateScript) [inline README.md:190]: Using "Modify" might be better that"Control" ?

**Review by ruinmessi — COMMENTED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (Joker316701882) [reply README.md:190]: This tutorial covers two topics:

1. using the default size
2. using modified sizes

''Modify'' only stands for case 2. So I think ''control'' is a better description. What do you think?

**Pull request merged**